src/symbols.js

```diff
@@ -21,12 +21,16 @@ module.exports = {
     text: {},
 };
 
-function defineSymbol(mode, font, group, replace, name) {
+function defineSymbol(mode, font, group, replace, name, acceptUnicodeChar) {
     module.exports[mode][name] = {
         font: font,
         group: group,
         replace: replace,
     };
+
+    if (acceptUnicodeChar) {
+        module.exports[mode][replace] = module.exports[mode][name];
+    }
```

Comment:
This should extend pretty easily to other unicode characters that are simple matches.